### ISSUE TITLE
A11y_code.visualstudio.com_VisualStudioCode_HomePage_Screenreader: Screenreader reads as image for the links present in the Carousel.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode-docs#3699]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.3.1;#MAS4.1.2;#Win10;#Visual Studio Code;#Narrator;#NVDA; #CAITestPass_May’20;

Environment Details:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.488.1 (Official build) dev (64-bit)

Repro Steps:
1)Launch the application URL :
https://code.visualstudio.com/
2)Navigate to the image present in the carousel.
3)Open screenreader and check whether it reads as link which works as link.

Actual:
While navigating through keyboard, Screenreader reads image which reads as link.

Expected:
Screenreader should read as link for the images (Ex: Una Kravets image)
Role should be provided as link as it works as link.
User Impact:
Screenreader user unable to get the correct control type of the control.
Attachment link for Reference:
[3699_A11y_code.visualstudio.com_VisualStudioCode_HomePage_Screenreader.zip](https://github.com/microsoft/vscode-docs/files/4677592/3699_A11y_code.visualstudio.com_VisualStudioCode_HomePage_Screenreader.zip)

**Comment by gregvanl** [comment]
Fixed by changing role to link.

![image](https://user-images.githubusercontent.com/12818376/82931045-27edbc80-9f3b-11ea-913b-db1e2afa6176.png)

**Comment by ManishaC1** [comment]
Verified the issue in below environment.
URL: https://code.visualstudio.com/
Issue is fixed.
Github tags:#Closed
![Screenshot (436)](https://user-images.githubusercontent.com/105198219/217171133-10dd8a72-9a4e-45b2-b742-995bebd0c8f2.png)
